File: js/addresses.js
let TapyrusApi = require('tapyrus_api');

// Set your Access Token
let accessToken = ""

let defaultClient = TapyrusApi.ApiClient.instance;
defaultClient.defaultHeaders = { Authorization: `Bearer ${accessToken}` }

// You can change the host name and port number
// defaultClient.basePath = 'http://localhost:3000/api/v1'

let api = new TapyrusApi.AddressApi();

function addAddress() {
  return new Promise((resolve, reject) => {
    api.address((err, data, response) => {
      if (err) reject(err);
      resolve(data);
    })
  })
}

function getAddresses(opts) {
return new Promise((resolve, reject) => {
  api.addresses(opts, (err, data) => {
    if (err) reject(err);
    resolve(data);
  })
})
}

// @route   POST /api/v1/addresses 
// @desc    Generate new Address
addAddress()
  .then(data => console.log(data))
  .catch(err => console.log(err))

// @route   GET /api/v1/addresses 
// @desc    Get the list of addresses
let opts = { per: 10, page: 1 };
getAddresses(opts)
  .then(data => console.log(data))
  .catch(err => console.log(err))

File: java/addresses.java
import org.openapitools.client.ApiClient;
import org.openapitools.client.ApiException;
import org.openapitools.client.Configuration;
import org.openapitools.client.auth.*;
import org.openapitools.client.model.*;
import org.openapitools.client.api.AddressApi;

public class Addresses {
    public static void main(String[] args) {
        String accessToken = "";

        ApiClient defaultClient = Configuration.getDefaultApiClient();
        defaultClient.setBasePath("http://localhost:3000/api/v1");
        defaultClient.addDefaultHeader("Authorization", "Bearer " + accessToken);

        AddressApi apiInstance = new AddressApi(defaultClient);
        try {
            String addAddressResponse = apiInstance.address();
            System.out.println(addAddressResponse);

            Integer per = 10;
            Integer page = 1;
            GetAddressesResponse getAddressesResponse = apiInstance.getAddresses(per, page);
            System.out.println(getAddressesResponse);
        } catch (ApiException e) {
            System.err.println("Exception when calling AddressApi#getAddresses");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}